tool/gravity/cli/update.go

```diff
@@ -73,21 +85,12 @@ func updateTrigger(
 		return trace.Wrap(err, "failed to connect to teleport proxy")
 	}
 
-	clusterEnv, err := localEnv.NewClusterEnvironment()
-	if err != nil {
-		return trace.Wrap(err)
-	}
-
-	if clusterEnv.Client == nil {
-		return trace.BadParameter("this operation can only be executed on one of the master nodes")
-	}
-
 	app, err := checkForUpdate(localEnv, operator, cluster, appPackage)
 	if err != nil {
 		return trace.Wrap(err)
 	}
 
-	err = checkCanUpdate(*cluster)
+	err = checkCanUpdate(*cluster, operator, app.Manifest)
```

Comment:
I wonder - if you change storage driver, how does operation creation below work? Won't `operator.CreateSiteAppUpdateOperation` fail because the server still does not allow switching drivers?